### ISSUE TITLE
Error while trying to call a memoized function

### ISSUE THREAD

**Issue by abelcastilloavant** [peterhurford/s3memoize#2]
Can't memoize a function. Minimal example:
```
> library(s3memoize)
> options(error = bettertrace::stacktrace)
> f <- function(x) x^2
> f(3)
[1] 9
> g <- s3memoize(f)
> g
$before_fn
function (x)
x^2

$after_fn
{
    hash <- `_digest`(c(list(..., s3memoize.reload, s3memoize.verbose),
        lapply(`_additional`, function(x) eval(x[[2L]], environment(x)))),
        algo = "sha512")
    if (`_cache`$has_key(hash)) {
        res <- `_cache`$get(hash)
    }
    else {
        res <- withVisible(`_f`(... = ..., s3memoize.reload = s3memoize.reload,
            s3memoize.verbose = s3memoize.verbose))
        `_cache`$set(hash, res)
    }
    if (res$visible) {
        res$value
    }
    else {
        invisible(res$value)
    }
}

> g(3)
Error in eval(expr, envir, enclos) : '...' used in an incorrect context
In package s3memoize: g(3)
In package base: withVisible(`_f`(... = ..., s3memoize.reload = s3memoize.reload,      s3memoize.verbose = s3memoize.verbose))
In package s3memoize: `_f`(... = ..., s3memoize.reload = s3memoize.reload, s3memoize.verbose = s3memoize.verbose)
In package checkr: s3memoize::get_s3path(fn, ...)
In package base: lapply(as.list(sys.call()[-1]), function(expr) {     eval(expr, parent.frame(3)) })
In package checkr: FUN(X[[i]], ...)
In package base: eval(expr, parent.frame(3))
In package s3memoize: eval(expr, envir, enclos)

Error: '...' used in an incorrect context
> packageVersion('s3memoize')
[1] ‘0.2.0’
> packageVersion('checkr')
[1] ‘0.1.4’
```

R version info:
```
R version 3.2.4 (2016-03-10) -- "Very Secure Dishes"
Copyright (C) 2016 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin13.4.0 (64-bit)
```

**Comment by peterhurford** [comment]
I've replicated this bug. :(

**Comment by peterhurford** [comment]
This appears to be a bug with checkr. I have [reverted the offending checkr code as to not break this package](https://github.com/peterhurford/s3memoize/pull/3) and [filed a bug separately with checkr](https://github.com/peterhurford/checkr/issues/37). I'll update this when the PR is merged and a fixed version of s3memoize is available.

**Comment by peterhurford** [comment]
This should now be fixed in v0.2.1. Reopen this issue if the problem persists.
